### PR TITLE
feat(launchpad): improve layout and design

### PR DESCRIPTION
**Pull request by marktnoonan**
- Closes https://cypress-io.atlassian.net/browse/UNIFY-359

### Additional details

These are quick style wins that were able to be completed in the course of auditing the launchpad UI to identify remaining areas needing to get into alignment with Figma. There is more to do, but no reason not to take these steps now. The other output of UNIFY-359 will be a list of other UI tasks to complete or investigate more based on the audit.

### How has the user experience changed?

Lots of small style enhancements that bring us closer to 1:1 with the designs in Figma, for example: 

Before: 
launchpad - before

After:
launchpad - after

### CODE REVIEW

File: packages/launchpad/src/setup/TestingTypeCard.vue [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
     <h2 class="mt-4 text-primary text-18px">{{ title }}</h2>
-    <p class="mt-3 text-sm" v-html="description + ' Aenean lacinia bibendum nulla sed consectetur. Morbi leo risus, porta ac consectetur ac, vestibulum at eros. Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Aenean lacinia bibendum nulla sed.'" />
-    <div class="flex justify-between mt-4">
+    <p class="mt-3 text-sm mb-60px" v-html="description" />
+    <div class="absolute left-0 right-0 flex justify-between bottom-30px px-30px">
       <span class="flex items-center px-3 py-1 border rounded-full">
         <i-oi-clock class="mr-2 text-gray-300 w-14px h-14px" />8 min
       </span>
-      <span class="border rounded-full pt-2px pl-4px text-body-gray-400 w-32px h-32px">
+      <span class="border rounded-full text-body-gray-400 w-32px h-32px flex items-center justify-center">
```
Comment (JessicaSachs) [inline packages/launchpad/src/setup/TestingTypeCard.vue:16]: @marktnoonan the play button centering was off

**Review by JessicaSachs — APPROVED** [review]
I pulled this down and used it. This is a great improvement. The next steps for this page are to:
1. Sync w @ryanjwilke to do an in-depth review of the screens
2. Figure out responsive design. I would personally just wrap the cards with `flex-wrap` at a certain window width.

File: packages/launchpad/src/setup/TestingTypeCard.vue [hunk 2]
```diff
@@ -1,19 +1,19 @@
 <template>
   <div
-    class="flex flex-col w-full border border-gray-200 rounded p-30px text text-body-gray-500"
+    class="relative flex flex-col w-full border border-gray-200 rounded p-30px text text-body-gray-500"
     :role="role"
   >
     <div class="border h-152px">
       <!-- temp wrapper for icon -->
       <component :is="icon" class="mx-auto mt-4" />
```
[the remark this answers is not in the document]
Comment (marktnoonan) [reply packages/launchpad/src/setup/TestingTypeCard.vue:8]: yeah at the time I just matched the shape from figma, we can adjust

File: packages/launchpad/src/setup/TestingTypeCard.vue (hunk 1 above)
Comment (marktnoonan) [reply packages/launchpad/src/setup/TestingTypeCard.vue:16]: good fix 👍

**Pull request merged**